### PR TITLE
[Prometheus Receiver] Added test case to validate 'Untyped' metrics

### PR DESCRIPTION
**Pull request by PaurushGarg**
Note: This PR is dependent on prior PR#904 . Changes of this PR are in commit id
9f2b001d7861c8631f4ec96906d1444800d873e8 : **Adding testcase for untyped metrics**

**Description:** 
- Added test case to validate `untyped` metric with default Prometheus parser.
- Used `untyped` in testdata as metric type, instead of `unknown`. `unknown` was applicable for OpenMetrics parser.

**Link to tracking Issue:** <Issue number if applicable>
Issue: 6083

### CODE REVIEW

File: receiver/prometheusreceiver/metrics_receiver_labels_test.go [hunk 1]
```diff
@@ -18,6 +18,7 @@ import (
 	"testing"
 
 	"github.com/prometheus/prometheus/pkg/labels"
+	"github.com/stretchr/testify/require"
```
Comment (Aneurysm9) [inline receiver/prometheusreceiver/metrics_receiver_labels_test.go]: How is it that a dependency on require is being added here but uses of it are being removed?  Something is not right here.

Comment (PaurushGarg) [reply receiver/prometheusreceiver/metrics_receiver_labels_test.go]: Hi Anthony this is coming from Merge. I didnt need `require.Greater(t, len(rms), 0, "At least one resource metric should be present")` as `verifyNumScrapeResults(t, td, rms)` is doing the same thing in a better way. This `require` is coming from merge with `verifyLabelLimitTarget1` test, as it is used there. I will co-ordinate and comeback if this is not needed and update accordingly.

Comment (PaurushGarg) [reply receiver/prometheusreceiver/metrics_receiver_labels_test.go]: @Aneurysm9 thanks, Updated. Removed  `require.Greater(t, len(rms), 0, "At least one resource metric should be present")` from `verifyLabelLimitTarget1`

**Review by Aneurysm9 — APPROVED** [review]

**Comment by PaurushGarg** [comment]
Merged.

**Pull request closed without merging**